supervisor/shared/serial.c

```diff
@@ -260,52 +260,71 @@ char serial_read(void) {
     return -1;
 }
 
-bool serial_bytes_available(void) {
+uint32_t serial_bytes_available(void) {
     #if CIRCUITPY_USB_VENDOR
-    if (tud_vendor_connected() && tud_vendor_available() > 0) {
-        return true;
+    if (tud_vendor_connected()) {
+        uint32_t count = tud_vendor_available();
+        if (count > 0) {
+            return count;
+        }
```

Comment:
I think you probably want to add these all together too because you can read them all.

Reply:
Aha, that makes sense -- I was not thinking of multiple available inputs.